### ISSUE TITLE
Is there way to add a check-icon dynamically to the menu items?

### ISSUE THREAD

**Issue by NafiurSiddiqui** [shuding/nextra#1643]

**Comment by irangarcia** [comment]
Maybe [this](https://nextra.site/docs/docs-theme/theme-configuration#customize-sidebar-content) could help you

**Comment by NafiurSiddiqui** [comment]
> Maybe [this](https://nextra.site/docs/docs-theme/theme-configuration#customize-sidebar-content) could help you

Yes, I noticed that. It works. Thank you! Could you give me any hint on how could I conditionally change that icon from a JSON file? for instance, if something is TRUE then the color should GREEN?

**Comment by C-EO** [comment]
Probaly use a conditional (an if...else statement)

**Comment by NafiurSiddiqui** [comment]
> Probaly use a conditional (an if...else statement)

There is no way we could do that without modifying the _meta file. Meaning modifying and saving the file from the server. That would be an insnane amount of request each time someone completed an item from the list. This is a great template. It would be great if we could customize it more.

**Comment by irangarcia** [comment]
#1657, maybe this could help you

**Comment by irangarcia** [comment]
#1657 this will help

**Comment by C-EO** [comment]
@NafiurSiddiqui  to back up @irangarcia you can actually also do it in your `theme.config.tsx` file. (As documented on the Nextra site)

**Comment by NafiurSiddiqui** [comment]
> #1657 this will help

This helped. Thank you so much. I provided further feedback on this in #1657

**Comment by NafiurSiddiqui** [comment]
> @NafiurSiddiqui to back up @irangarcia you can actually also do it in your `theme.config.tsx` file. (As documented on the Nextra site)

Yes. Thank you. A little example on #1657 actually helped.
